frontend/src/pages/Landing/LandingPage.tsx

```diff
@@ -59,24 +59,28 @@ function LandingPage() {
                   className={styles.HeaderText}
                   variant="h1"
                 >
-                  Equity Forward
+                  Advancing Health Equity
                 </Typography>
                 <br />
               </Hidden>
               <Hidden smUp>
                 <Typography className={styles.HeaderTextMobile}>
-                  Equity Forward
+                  Advancing Health Equity
                 </Typography>
               </Hidden>
               <Typography className={styles.HeaderSubtext} variant="body1">
                 <p>
-                  We know that our communities are experiencing life or death
-                  situations due to the inequities, conditions and policies into
-                  which they are born, grow, learn, work and age.
+                  We know that the data we collect can be flawed and at times even 
+                  worsen health inequities many people face if not reported or 
```

Comment:
"worsen health inequities" --> "worsen the health inequities"?

Reply:
I just copied what I was given so I'm going to leave as is for now